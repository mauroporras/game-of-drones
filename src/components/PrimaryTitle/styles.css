.PrimaryTitle {
  color: var(--color-primary-text);
  margin: 0 0 2rem;
  text-align: center;
}
