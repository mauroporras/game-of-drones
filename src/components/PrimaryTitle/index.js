import React from 'react'

import './styles.css'

const PrimaryTitle = props => (
  <h1 className='PrimaryTitle'>{props.text}</h1>
)

export default PrimaryTitle
