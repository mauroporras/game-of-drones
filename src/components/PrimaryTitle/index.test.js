import React from 'react'
import renderer from 'react-test-renderer'

import PrimaryTitle from './index'

test("Component renders the correct text", () => {
  const component = renderer.create(
    <PrimaryTitle text='Hello' />
  )
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
