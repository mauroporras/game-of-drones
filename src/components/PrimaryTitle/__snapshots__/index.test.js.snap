exports[`test Component renders the correct text 1`] = `
<h1
  className="PrimaryTitle">
  Hello
</h1>
`;
